src/elements/core/styles/core.scss

```diff
@@ -160,6 +160,18 @@ img {
   object-position: var(--sbb-image-object-position);
 }
 
+// TODO: Move back to the sbb-container components when the global css refactoring happens
+sbb-container {
+  [slot='image']:is(sbb-image, img),
+  [slot='image'] :is(sbb-image, img) {
+    --sbb-image-object-fit: cover;
+
+    border-radius: var(--sbb-container-background-border-radius);
+    height: 100%;
+    position: absolute;
```

Comment:
I saw that in container.scss there is also the position:absolute. Are both needed in case it is a figure?

Reply:
Yes. Specifically, the `figure + img` case needs absolute on both (in theory it should not but I haven't understood why it behaves strangely. I've only been able to find the fix for it :/ )